.github/workflows/linter.yaml

```diff
@@ -55,15 +55,15 @@ jobs:
           VALIDATE_BASH: true
           SHELLCHECK_OPTS: "--severity=info"
 
-      # # TODO: remove comments when ready for markdown linting
-      # # set VALIDATE_ALL_CODEBASE to false before merging
-      # - name: Lint Markdown
-      #   if: ${{ success() || failure() }}
-      #   uses: github/super-linter/slim@v4
-      #   env:
-      #     VALIDATE_ALL_CODEBASE: true
-      #     DEFAULT_BRANCH: main
-      #     FILTER_REGEX_EXCLUDE: cli/.*
-      #     LINTER_RULES_PATH: .github/linter-rules
-      #     VALIDATE_MARKDOWN: true
-      #     MARKDOWN_CONFIG_FILE: .markdown-lint-config.yaml
+      # TODO: remove comments when ready for markdown linting
+      # set VALIDATE_ALL_CODEBASE to false before merging
```

Comment:
2 things here: 
- @alaincormier-ssc can you confirm that the value should be `false` in the block below
- I expect this line to be removed by this PR

Reply:
1. yes, `VALIDATE_ALL_CODEBASE` can be set to false before merging so that only modified content will be looked at in future PRs.  It was initially set to true to validate linting on content that already existed in the entire repo.
2. yes, comments on line 58 and 59 should be removed.
3. maybe something like this? `# super-linter configuration for Markdown files`